nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>Maybe it would be useful to write a little text before this code. Something like: </p><p>"Next we will set some parameters for our peak detection problem. Depending on your setting and goals this parameter might change. In this case we want to predict 1 peak (npeaks) and ...."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Done</p>